routes/clinical.js

```diff
@@ -10,7 +10,9 @@ var router = express.Router();
 // so that the user is automatically prompted to save the file from their browser.
 const apiRoot = urlJoin(CLINICAL_SERVICE_ROOT, SUBMISSION_TEMPLATE_PATH);
 
-// This is mostly passthrough, so file names and response should be set in clincial
+// This is mostly passthrough, so file names and response will be set in clincial
```

Comment:
@hlminh2000 not sure if this affects you right now...

Reply:
sounds backward compatible so shouldn't be a breaking change for me. Though does this explanation appear in swagger somewhere? :P

Reply:
no lol, guess I could put in something

Reply:
yeah I just figured as a consumer of the API I'd prefer to find this information there than digging into source code :P thanks!

Reply:
I removed the all.zip it was more confusing